sdk/iot/Azure.Iot.Hub.Service/samples/IotHubClientSamples/ModuleIdentityLifecycleSamples.cs

```diff
@@ -24,40 +26,37 @@ public async Task RunSampleAsync()
             string moduleId = $"module{Random.Next(MaxRandomValue)}";
 
             // Create a DeviceIdentity.
-            DeviceIdentity deviceIdentity = await CreateDeviceIdentityAsync(deviceId);
+            await CreateDeviceIdentityAsync(deviceId);
 
             // Create a ModuleIdentity.
-            ModuleIdentity moduleIdentity = await CreateModuleIdentityAsync(deviceIdentity, moduleId);
+            await CreateModuleIdentityAsync(deviceId, moduleId);
 
             // List all modules within the device.
-            await ListAllModulesAsync();
+            await ListAllModulesAsync(deviceId);
 
             // Get the Module Identity.
-            await GetModuleIdentityAsync();
+            await GetModuleIdentityAsync(deviceId, moduleId);
 
             // Update Module Identity.
-            await UpdateModuleIdentityAsync();
+            await UpdateModuleIdentityAsync(deviceId, moduleId);
 
             // Get Module Twin,
-            await GetModuleTwinAsync();
+            await GetModuleTwinAsync(deviceId, moduleId);
 
             // Update Module Twin.
-            await UpdateModuleTwinAsync();
-
-            // Invoke a method on the ModuleTwin
-            await InvokeMethodAsync();
+            await UpdateModuleTwinAsync(deviceId, moduleId);
 
             // Delete the module identity
-            await DeleteModuleIdentityAsync(moduleIdentity);
+            await DeleteModuleIdentityAsync(deviceId, moduleId);
 
             // Delete the device (cleanup)
-            await DeleteDeviceIdentityAsync(deviceIdentity);
+            await DeleteDeviceIdentityAsync(deviceId);
         }
 
         /// <summary>
         /// Creates a new device identity
         /// </summary>
-        /// <param name="deviceId">Device Id for the device identity</param>
+        /// <param name="deviceId">Device identity unique identifier</param>
```

Comment:
Don't have a strong opinion, just feel this can be simplify to may be "Unique identifier of a device".

Reply:
I don't feel strongly about it either. will change.